### ISSUE TITLE
2019-11-20 M2SD model execution with tc013 in application directly (SDEID: 343222739) fails with Error: $ operator is invalid for atomic vectors

### ISSUE THREAD

**Issue by tensfeldt** [tensfeldt/openNCA#48]
Execution of M2 SD bolus model with commit 92ca12a 

It appears that the 343222739 SDEID is associated with records where PKCNCN is ALL missing, i.e. NA. This needs to be addressed in a way that missing PKCNCN values do not impair execution or processing of profiles and generation of parameters for other profiles.

Links to #38 

```
There were 25 warnings (use warnings() to see them)
[1] "openNCA: Model: M2 DosingType: SD AUCMETHOD: LINLOG"
Error in value[[3L]](cond) : 
Error: $ operator is invalid for atomic vectors
For SDEID 343222739
Calls: run_computation ... tryCatch -> tryCatchList -> tryCatchOne -> <Anonymous>
In addition: Warning messages:
1: In run_M2_SD_computation(data = merged_data, map = map_data, method = method, :
Flag 'FLGEMESIS' is not present in the dataset
2: In run_M2_SD_computation(data = merged_data, map = map_data, method = method, :
Flag 'FLGACCEPTTAU' is not present in the dataset
Execution halted

```

**Comment by jhhughes256** [comment]
This is also linked to #27 where tc013 (also M2SD) provides a similar error.

**Comment by tensfeldt** [comment]
#48 appears to be resolved with commit 96f6417 so closing #48. Other issues remain with missing interpolated concentration values for partial AUC0_20 which will be reported separately.
